README.md

```diff
@@ -10,50 +10,84 @@
   <img src="http://www.introspectivesystems.com/wp-content/uploads/2017/12/post-xGraph-medium-570x350.png" alt="Sublime's custom image"/>
 </p>
 
+v1.1.1
+
 Introspective Systems, LLC.
 
 ---
-### xGraph Executable
-The xGraph command line interface accelerates the building and running
-of xGraph Systems.
 
----
-#### Getting Started
-You will find all the necessary installation files in the `.\bin\`
-directory. To install the xgraph CLI, follow the instructions for your
-operating system found below.
+The xGraph package comes with two main features: the xGraph core and the xGraph 
+Command Line Interface.
```

Comment:
I would only include references to the Command Line utility here. The only people who need to care about nexus/genesis are those who are using xgraph programmatically.

As well, calling the files on their own is also no longer supported.

Reply:
I disagree. This is the README.md for the core, so it is what shows up on our git homepage, so we should explain what it is. Once we have a website, we can provide better user-end instructions.

What do you mean calling files on their own is no longer supported?

Reply:
I agree, that we should provide all the relevant information here about what xgraph is, but "Nexus" and "Genesis" arent really public facing anymore, they're merely an implementation detail of the command line utility. The end-user will never even see the words nexus or genesis if we've done our job well.

I mean exactly that, you can no longer call node directly on nexus or genesis. they are not executable files. they are effectively two libraries that xgraph uses to create systems